src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/bootstrap_4_horizontal_layout.html.twig

```diff
@@ -94,6 +94,10 @@
     col-sm-10
 {%- endblock form_group_class %}
 
+{% block form_row_class -%}
```

Comment:
why do you need to do that in this pull request?

Reply:
so that I can override it in my module, I originally even wanted to change default class with `form-group row` but I don't want to risk breaking the other pages